Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Settings/CarlaSettingsDelegate.cpp

```diff
@@ -158,7 +158,7 @@ void UCarlaSettingsDelegate::CheckCarlaSettings(UWorld *world)
   {
     return;
   }
-  if (world == nullptr || !IsValid(world) || world->IsPendingKill())
+  if (world == nullptr || !IsValid(world) )
```

Comment:
```suggestion
  if (IsValid(world))
```
IsValid() already checks if world is null